proposals/lip-0074.md

```diff
@@ -904,6 +905,10 @@ def applyProposalOutcome(type: int, data: bytes) -> None:
     if type == PROPOSAL_TYPE_FUNDING:
         # assume that the data is valid
         dataObj = decode(fundingProposalDataSchema, data)
+        # lock donated treasury tokens together with minted treasury tokens
+        treasuryDonations = Token.getAvailableBalance(ADDRESS_TREASURY, configStore.tokenIdTreasury)
+        if (treasuryDonations != 0):
+            Token.lock(ADDRESS_TREASURY, MODULE_NAME_GOVERNANCE, configStore.tokenIdTreasury, treasuryDonations)
```

Comment:
I see you decided to lock the donated tokens, so they become undistinguishable from the minted tokens. I can't think of any reason why this shouldn't be so, and it seems like a fine solution to me. 
I suppose adding a comment to the Treasury part of the Rationale stating that users can freely donate to treasury is unnecessary, right?

Reply:
Have added a sentence to the rationale.